flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/config/ExecutionConfigOptions.java

```diff
@@ -100,29 +100,41 @@
 					"default parallelism is set, then it will fallback to use the parallelism " +
 					"of StreamExecutionEnvironment.");
 
+	private static final String MEMORY_DESCRIPTION = "Note: memory size is only a weight hint," +
```

Comment:
don't do this...

Reply:
You mean just repetitively append this msg to memory config options?

Reply:
yes

Reply:
First time I think it is for generating document, so after generating, the document is good to be read.
But we can make it is good to be read too.